web3_proxy/src/frontend/errors.rs

```diff
@@ -96,6 +182,78 @@ impl FrontendErrorResponse {
                     ),
                 )
             }
+            Self::EipVerificationFailed(err_1, err_191) => {
+                warn!(
+                    "EipVerificationFailed err_1={:#?} err2={:#?}",
+                    err_1, err_191
+                );
+                (
+                    StatusCode::UNAUTHORIZED,
+                    JsonRpcForwardedResponse::from_string(
+                        format!(
+                            "both the primary and eip191 verification failed: {:#?}; {:#?}",
+                            err_1, err_191
+                        ),
+                        Some(StatusCode::UNAUTHORIZED.as_u16().into()),
+                        None,
+                    ),
+                )
+            }
+            Self::EthersHttpClientError(err) => {
+                warn!("EthersHttpClientError err={:?}", err);
+                (
+                    StatusCode::INTERNAL_SERVER_ERROR,
+                    JsonRpcForwardedResponse::from_str(
+                        "ether http client error",
+                        Some(StatusCode::INTERNAL_SERVER_ERROR.as_u16().into()),
+                        None,
+                    ),
+                )
+            }
+            Self::EthersProviderError(err) => {
+                warn!("EthersProviderError err={:?}", err);
+                (
+                    StatusCode::INTERNAL_SERVER_ERROR,
+                    JsonRpcForwardedResponse::from_str(
+                        "ether provider error",
+                        Some(StatusCode::INTERNAL_SERVER_ERROR.as_u16().into()),
+                        None,
+                    ),
+                )
+            }
+            Self::EthersWsClientError(err) => {
+                warn!("EthersWsClientError err={:?}", err);
+                (
+                    StatusCode::INTERNAL_SERVER_ERROR,
+                    JsonRpcForwardedResponse::from_str(
+                        "ether ws client error",
+                        Some(StatusCode::INTERNAL_SERVER_ERROR.as_u16().into()),
+                        None,
+                    ),
+                )
+            }
+            Self::FlumeRecvError(err) => {
+                warn!("FlumeRecvError err={:#?}", err);
+                (
+                    StatusCode::INTERNAL_SERVER_ERROR,
+                    JsonRpcForwardedResponse::from_str(
+                        "flume recv error!",
+                        Some(StatusCode::INTERNAL_SERVER_ERROR.as_u16().into()),
+                        None,
+                    ),
+                )
+            }
+            Self::GasEstimateNotU256 => {
+                warn!("GasEstimateNotU256");
+                (
+                    StatusCode::BAD_REQUEST,
```

Comment:
i think this is an application error and not a warn/BAD_REQUEST. It means the response from one of our backend servers was broken somehow.